packages/atomic/tailwind.config.js

```diff
@@ -27,6 +27,13 @@ module.exports = {
         md: 'var(--atomic-border-radius-md)',
         lg: 'var(--atomic-border-radius-lg)',
         xl: 'var(--atomic-border-radius-xl)',
+        //left-rounded
+        'lr': 'var(--atomic-border-radius) 0px 0px var(--atomic-border-radius)',
+        //right-rounded
+        'rr': '0px var(--atomic-border-radius) var(--atomic-border-radius) 0px',
```

Comment:
you should be able to use this instead of creating new styles https://tailwindcss.com/docs/border-radius#rounding-sides-separately

eg. `rounded-r`